src/rail/creation/degradation/observing_condition_degrader.py

```diff
@@ -358,7 +387,11 @@ def apply_galactic_extinction(self, pixel: int, pixel_cat: pd.DataFrame) -> pd.D
         ebvvec = self.maps["EBV"][ind]
 
         if "renameDict" in self.maps.keys():
-            bands = self.maps["renameDict"]
+            # in case renameDict contains other things than bands
+            bands={}
+            for b in ["u","g","r","i","z","y"]:
```

Comment:
nit: It might be nice to turn this into a constant at the top of the module. Something like `STANDARD_BANDS = ["u", "g", ...]`. You could potentially also make `band_a_ebv` a constant.